README.md

```diff
@@ -599,3 +604,23 @@ deploy               Deploy wheel to public s3 bucket
 clean                Clean hidden and compiled files
 help                 Show help message
 ```
+
+## 🔌 How to use production models
+
+We'll go through an example on how to download data and models, and use them for prediction and training using the cli commands.
+
+Data can be downloaded from here after you make an account: http://participants-area.bioasq.org/datasets/
+Download from the `Datasets for task a` section, the latest available txt format dataset. This will result in a zip file what you can unzip, preferably in the `data\raw` folder. 
+You can download pretrained model from the releases section of this repository. For the current example we'll a model trained with the `mesh-cnn` approach: https://github.com/wellcometrust/grants_tagger/releases/download/v0.1.3/disease_mesh_cnn-2021.03.1.tar.gz
```

Comment:
Note that the latest model could not be added to a release cause it was bigger than 2GB. So the standard way to download a model would be to use the CLI `grants_tagger download mesh`. That said assuming someone just wants to download the model we should add link that is updated in the README that either links to the release URL or s3, the link is the one in the code to download the model, this is just a manual way to get it.

Reply:
problem here is that it seems you need credentials to download from s3? I tried here to offer the public version.

Reply:
you should not need credentials so if you do, I need to fix that. its shared in a public bucket

Reply:
yeah, seems this worked:

Reply:
`grants_tagger download epmc-mesh models/`

Reply:
`grants_tagger download model mesh`
seems to have worked, so no need. I'll update the README